scripts/SREP/etcd-complete-health-check/metadata.yaml

```diff
@@ -0,0 +1,48 @@
+file: script.sh
+name: etcd-complete-health-check
+description: Prints out the complete health check for etcd with recent logs incase of unhealthy clusters
+author: devppratik 
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
+  - MTSRE
+labels:
+  - key: OSD_TYPES
+    description: Compatible cluster types for this script
+    values:
+      - OSD
+      - HyperShift
```

Comment:
Is this necessarily true? I'm not 100% clear on how this key is enforced, but if this were to run on a HyperShift hosted cluster it's not going to work, nor does it have logic on how to get this from a HCP Namespace.